docs/user-guides/finetuner.md

```diff
@@ -0,0 +1,144 @@
+# Fine-tuning CLIP Models Using Finetuner
```

Comment:
i would make it a bit shorter, see 
![image](https://user-images.githubusercontent.com/9794489/179534219-46c76f77-51d7-4d69-bdee-e2a983f219e2.png)

Reply:
use `Fine-tune a pretrained model` as the title? @ZiniuYu you can also borrow some idea from this doc https://huggingface.co/docs/transformers/training

Reply:
I will go with Fine-tune Models